ui-tests-starter/tst-243+/software/aws/toolkits/jetbrains/uitests/OfflineAmazonQInlineCompletionTest.kt

```diff
@@ -28,6 +30,7 @@ class OfflineAmazonQInlineCompletionTest {
         di = DI {
             extend(di)
             bindSingleton<CIServer>(overrides = true) { TestCIServer }
+            ConfigurationStorage.Companion.logEnvironmentVariables(false)
```

Comment:
we should override the defaults so reset() does not flip this back

Reply:
I tried doing that but the ConfigurationStorage class which has the default values cannot be directly inherited from...

Reply:
try
```
            val defaults = ConfigurationStorage.instance().defaults.toMutableMap().apply {
                put("LOG_ENVIRONMENT_VARIABLES", "false")
            }

            bindSingleton<ConfigurationStorage>(overrides = true) {
                ConfigurationStorage(this, defaults)
            }
```

Reply:
Yep, sorry, I could access the env vars only in 243, I think I was trying in 242 earlier. The var is public